Carnap/src/Carnap/Languages/PurePropositional/Logic/Ripley.hs

```diff
@@ -71,7 +71,8 @@ parseRipleyLNJ rtc n _ = do
                            , "&I" , "/\\I" , "∧I"
                            , "&EL", "/\\EL", "∧EL"
                            , "&ER", "/\\ER", "∧ER"
-                           , "->I", "→I", "->Iv"
+                           , "->Iv"
```

Comment:
This handles a quirk of the parsec parser. Parsec checks these strings in sequence, and returns the first one which occurs as an *initial segment* of the string being parsed. So if "->I" is listed before "->Iv", then "->Iv" will never be returned.